roles/kubernetes-apps/csi_driver/vsphere/templates/vsphere-csi-controller-rbac.yml.j2

```diff
@@ -40,7 +40,7 @@ rules:
     verbs: ["get", "list", "watch"]
   - apiGroups: ["storage.k8s.io"]
     resources: ["volumeattachments"]
-    verbs: ["get", "list", "watch", "patch"]
+    verbs: ["get", "list", "watch", "create", "update", "delete", "patch"]
```

Comment:
According to https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/3a71f97f4dd61e770ddc4d6492f3e4d9522907b1/manifests/guestcluster/1.22/pvcsi.yaml#L45-L46
the upstream manifest is
```
resources: ["volumeattachments"]
verbs: ["get", "list", "watch", "update", "patch"]
```
So I guess it would be fine to add `"update"` only.
Isn't it enough for fixing the issue?

Reply:
Hi @oomichi I guess it could work. Unfortunately, I don't have access anymore to the vSphere environment to test.

Reply:
OK, how about adding `"update"` only for syncing to the upstream vsphere-csi-driver?
If you still face the issue, we would update the upstream manifest first, then the change will be able to be applied after that.
Through this process, we can get good technical feedback from vsphere-csi-driver maintainers.

Reply:
@oomichi That sounds good.